src/App.js

```diff
@@ -4,40 +4,27 @@ import './App.css';
 
 const App = () => {
 
-const api_key = 'EW43splo4ni8vWJ6oN2cJunf49yeRxy6IFk4aOTb';
-const[search, setSearch] = useState('');
-const [data, setdata] = useState([]);
-
-// const [query, setQuery] = useState('yello');
-
-
-useEffect(() => {
-  getParks();
-
-});
-
-const getParks = async () => {
-  const response = await fetch(`https://developer.nps.gov/api/v1/parks?parkCode=acad&api_key=${api_key}`)
-  // const response = await fetch (`https://developer.nps.gov/api/v1/activities?id=hiking&q=q&limit=20&start=0&api_key=EW43splo4ni8vWJ6oN2cJunf49yeRxy6IFk4aOTb`)
-  const data = await response.json();
-  
-  
-  
-  console.log(data);
-  
-  
-};
-
-const updatedSearch = e =>{ 
-  setSearch(e.target.value);
-
-}
-// const getSearch = e => {
-//   e.preventDefault();
-//   setQuery(search);
-//   getSearch('');
-// }
-
+  // const api_key = 'EW43splo4ni8vWJ6oN2cJunf49yeRxy6IFk4aOTb';
+  const api_key = 'iBIDgu279HOLibT9B72MGBhW3NOT8PIbD92dfSIr';
+  const [search, setSearch] = useState('');
+  const [parks, setParks] = useState([]);
+
+  useEffect(() => {
+    getParks();
+  }, []);
+
+  const getParks = async () => {
+    const response = await fetch(`https://developer.nps.gov/api/v1/parks?api_key=${api_key}`)
+    // const response = await fetch(`https://developer.nps.gov/api/v1/parks?parkCode=acad&api_key=${api_key}`)
+    // const response = await fetch (`https://developer.nps.gov/api/v1/activities?id=hiking&q=q&limit=20&start=0&api_key=EW43splo4ni8vWJ6oN2cJunf49yeRxy6IFk4aOTb`)
+    const json = await response.json();
+    console.log(json);
+    setParks(json.data);
```

Comment:
you were really close with your code! the confusing part was that the json response from their API endpoint was actually shaped this like:
```
{
  start: 0,
  limit: 50.
  total: 467,
  data: [{}, {}, ...] // the data field was an array of objects - each object representing a park
}

So in your previous code, you would have had to access the park info by using `data.data`, so I renamed the json part to `json` to make it more clear, and then we access the parks info by `json.data`